### PR TITLE
[15.0] date_range Migration to V15

### PR DESCRIPTION
**Pull request by alan196**
### Proposed changes

Migration PR for module date_range.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the necessary documentation

### CODE REVIEW

**Review by JoanMForgeFlow — CHANGES_REQUESTED** [review]
Tested Locally, not working: In the CustomFilter --> Dates, my dataranges that were created earlier are not showing up. This is due to the following error: (see the attached image).
 
Captura de pantalla de 2021-11-10 13-37-50
Captura de pantalla de 2021-11-10 13-38-35
Captura de pantalla de 2021-11-10 13-39-22

File: date_range/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Date Range",
+    "summary": "Manage all kind of date range",
+    "version": "15.0.1.0.1",
```
Comment (BernatPForgeFlow) [inline date_range/__manifest__.py:6]: 15.0.1.0.0

**Comment by AaronHForgeFlow** [comment]
ping @alan196 are you plan to continue?

**Comment by alan196** [comment]
I'm really sorry, right now I'm busy and I cannot continue until 2 more weeks @AaronHForgeFlow, you can continue if you want and I close this MR 😞

**Comment by pedrobaeza** [comment]
Superseded by #414

**Pull request closed without merging**